lib/player.rb

```diff
@@ -11,7 +13,7 @@ def take_turn
     raise NotImplementedError, "A player needs to be able to take a turn"
   end
 
-  def render_board
-    @board.render
+  def render_board(show_ship = false)
+    @board.render(show_ship)
```

Comment:
I think this was part of a different issue. I attached that issue with this PR as well but next I'd recommend trying to separate your tasks.